test/api/Animated.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import View from '../../src/components/View.js';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import Animated from '../../src/api/Animated';
+
+describe.only('Animated.View', () => {
+  before(() => {
+    const jsdom = require('jsdom').jsdom;
+    global.document = jsdom('');
+    global.window = document.defaultView;
+    Object.keys(document.defaultView).forEach((property) => {
+      if (typeof global[property] === 'undefined') {
+        global[property] = document.defaultView[property];
+      }
+    });
+
+    global.navigator = {
+      userAgent: 'node.js',
+    };
+  });
```

Comment:
Yeah, this is hacky. I was focused on getting these changes in quickly because they are blocking my Kyle and Jimmy, but it's probably worth pulling this out while we are at it. If we don't we should definitely pull out the global setup when we write mount test #2.